raven/utilities/forecasting.py

```diff
@@ -0,0 +1,151 @@
+"""
+Tools for hydrological forecasting
+"""
+
+from raven.models import get_model
+import logging
+import xarray as xr
+import numpy as np
+import pandas as pd
+import datetime as dt
+import tempfile
+import pdb
+
+LOGGER = logging.getLogger("PYWPS")
+
+# This function gets model states after running the model (i.e. states at the end of the run).
+def getRavenStates(model, **kwds):
```

Comment:
Python style suggest to use lower case for functions, e.g. `ge_raven_states`, and CamelCase for classes.

Reply:
changed, thanks!